src/App.tsx

```diff
@@ -73,8 +81,11 @@ const App = () => {
                     setTomorrowTodo((t) => [...tomorrowTodo, todo]);
                     return;
                 case DATE.THIS_WEEK:
-                    setThisWeekTodo((t) => [...thisWeekTodo, todo]);
-                    return;
+                    setThisWeekTodo((thisWeekTodo) => [...thisWeekTodo, todo]);
+                    break;
+                case DATE.THIS_MONTH:
+                    setThisMonthTodo((thisMonthTodo) => [...thisMonthTodo, todo]);
+                    break;
```

Comment:
`break` runs the bottom code after the `switch` `return` will immediately go out of the function, which is better in this case, since there is no code after the `switch` we want to run